client/script.js

```diff
@@ -1,24 +1,31 @@
-'use strict'
-
-const ShowPassword = (function (my) {
-
-  my.toggle = function(selector) {
-    const elem = document.querySelector(selector)
-    elem.type = (elem.type === 'password') ? 'text' : 'password'
-  }
-
-  return my
-
-}) ({})
-
-const Display = (function (my) {
-
-  my.toggle = function (selector) {
-    document.querySelectorAll(selector).forEach(elem => {
-      elem.style.display = (elem.style.display === 'none') ? 'block' : 'none'
-    })
-  }
-
-  return my
-
-}) ({})
+import '@first-lego-league/user-interface/current/assets/css/app.css'
+import './stylesheet.css'
+
+const PASSWORD_FIELD_ID = 'password-field'
+const PASSWORD_BUTTONS_CLASS = 'password-button'
+
+function onReady () {
+  const showPassword = () => {
```

Comment:
Call this `togglePasswordVisibility` and not `showPassword`. It not only shows the password.